stable/kong/README.md

```diff
@@ -324,6 +324,15 @@ If your SMTP server requires authentication, you should the `username` and
 `smtp_password_secret` must be a Secret containing an `smtp_password` key whose
 value is your SMTP password.
 
+### Declarative Configuration
+
+The helm chart assumes to be running in declarative mode when it is started in 
+DB-less mode (`env.database: "off"`) in combination with no Kong ingress controller 
```

Comment:
nit:

when it is started in DB-less mode (`env.database: "off"`) and without the Kong ingress controller